bots/csr/src/test/java/org/openjdk/skara/bots/csr/CSRBotTests.java

```diff
@@ -39,6 +39,9 @@
 import static org.junit.jupiter.api.Assertions.*;
 
 class CSRBotTests {
+    private final static String CSR_UPDATE_MARKER = "<!-- csr: 'update' -->";
+    private static final String PROGRESS_MARKER = "<!-- Anything below this marker will be automatically updated, please do not edit manually! -->";
```

Comment:
I didn't notice this before, but these markers shouldn't be redefined here in the test. Make them package protected or even public in `PullRequestWorkItem` instead.

Reply:
> 

Sure, will fix it. Thx!